SharpPcap/SharpPcap.csproj

```diff
@@ -7,7 +7,7 @@ SPDX-License-Identifier: MIT
 -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net8.0</TargetFrameworks>
```

Comment:
request to add `net6.0`

Reply:
`netstandard2.0` means that net6.0 is also supported, however trimming support remains exclusive to .NET 8

Reply:
Well, yes. But `netstandard2.0` also means that .NET 8 is supported? Why limit the trimming to .NET 8? I've tried with 
`<TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>`

Reply:
yes, I would like to reduce the maintenance effort needed for an extra target framework, especially since it is going out of support soon.

It may look like we just need to add the framework in the project Metadata, but in reality we also need to add it to CI scripts and automated tests, and I would like to avoid that.